server/migrations/20210914210024-clean-up-db.js

```diff
@@ -2,6 +2,7 @@
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
+    return;
```

Comment:
What's with this extra return?

Reply:
Oops, this was erroring for me when trying to run migrations, thought I hadn't saved or committed this, good catch